tst/EnergyPlus/api/TestDataTransfer.c

```diff
@@ -70,6 +70,10 @@ void afterZoneTimeStepHandler(EnergyPlusState state)
             exit(1);
         }
         handlesRetrieved = 1;
+        unsigned int arraySize;
+        struct APIDataEntry * data = getAPIData(state, &arraySize);
+        int x = 1;
+        freeAPIData(data, arraySize);
```

Comment:
I'll comment this out in a minute, but this is how it would be used in the C API, not that I expect very many folks to do it that way.